src/commands/mock.ts

```diff
@@ -0,0 +1,51 @@
+import { CommandInteraction, SlashCommandBuilder } from 'discord.js';
+import { mockTargets } from '../index';
+
+export const Mock = {
+    info: new SlashCommandBuilder()
+        .setName('mock')
+        .setDescription('Mocks designated member')
+        .addUserOption(option =>
+            option.setName('member')
+                .setDescription('Choose which member you want to be mocked')
+                .setRequired(true)),
+
+    run: async (interaction: CommandInteraction): Promise<void> => {
+        if (!interaction.guild) { //Checks if the command is NOT done in a server
+            await interaction.reply('This command can only be done in a server.');
+            return;
+        }
+        //Looks for 'staff' role in server (case insensitive)     
+        const role = interaction.guild.roles.cache.find(role => role.name.toLowerCase() === 'staff');
+
+        if (!role) {
+            await interaction.reply('There is no \"staff\" role on this server.' + 
```

Comment:
If you're using single quotes to represent the string, you don't need to escape the quotation mark.

```suggestion
            await interaction.reply('There is no "staff" role on this server.' + 
```